app/controllers/users_controller.rb

```diff
@@ -13,6 +25,6 @@ def set_user
 
   # strong params
   def user_params
-    params.require(:user).permit(:name, :bio)
+    params.require(:user).permit(:name, :bio, :avatar)
```

Comment:
When I say not a blocker, I mean you can keep it the way you have it if you want. It's your call.